openpower/patches/p8dtu-patches/hostboot-p8/0008-Fill-board-mfg-time-and-date-in-FRU3.-It-is-read-fro.patch

```diff
@@ -0,0 +1,99 @@
+From ed60d5736559c4e100e5db20de942a584b58b6f0 Mon Sep 17 00:00:00 2001
+From: Jim Yuan <jim.yuan@supermicro.com>
+Date: Wed, 25 May 2016 15:24:40 -0700
+Subject: [PATCH 08/14] Fill board mfg time and date in FRU3. It is read from
+ PVPD:VNDR:IN.
+
+Signed-off-by: Jim Yuan <jim.yuan@supermicro.com>
+---
+ src/usr/ipmiext/ipmifruinv.C | 65 +++++++++++++++++++++++++++++++++-----------
+ 1 file changed, 49 insertions(+), 16 deletions(-)
+
+diff --git a/src/usr/ipmiext/ipmifruinv.C b/src/usr/ipmiext/ipmifruinv.C
+index 23aaf9b97..11fbd7677 100644
+--- a/src/usr/ipmiext/ipmifruinv.C
++++ b/src/usr/ipmiext/ipmifruinv.C
+@@ -1279,31 +1279,64 @@ errlHndl_t backplaneIpmiFruInv::buildChassisInfoArea(
+     return l_errl;
+ }
+ 
++// Quick hexdigit to binary converter.
++// Hopefull someday to replaced by strtoul
++uint8_t aschex2bin(char c)
```

Comment:
hostboot doesn't have strtoul?

Reply:
This is working right now.  can we just let it merge and then open an issue to address the additional change?

Reply:
strtoul wasn't added until the p9 stream (late last year actually)

Reply:
It seems to have been added as part of this commit. I'll leave it up to @dcrowell77 on if he'd prefer a part cherry-picked patch to bring in strtoul or to carry this patch as-is.
```
commit 21f75b9e4475b92665e4dd8ca182108dab53045f
Author: Ilya Smirnov <ismirno@us.ibm.com>
Date:   Wed Dec 12 11:33:38 2018 -0600

    SMF: NVRAM Reading and Mem Distribution end-to-end Changes
    
    This commit introduces the changes to read out the SMF secure
    memory amount value from NVRAM and to distribute the secure
    memory amount based on the value read. strtou64 was copied
    from runtime code to convert the value read from NVRAM (as a
    string) to uint64_t.
    
    Change-Id: I83e41f0aaff9b4035d20a517cf866f348acedd59
    Reviewed-on: http://rchgit01.rchland.ibm.com/gerrit1/69728
    Tested-by: Jenkins Server <pfd-jenkins+hostboot@us.ibm.com>
    Tested-by: Jenkins OP Build CI <op-jenkins+hostboot@us.ibm.com>
    Tested-by: FSP CI Jenkins <fsp-CI-jenkins+hostboot@us.ibm.com>
    Reviewed-by: Nicholas E. Bofferding <bofferdn@us.ibm.com>
    Tested-by: Jenkins OP HW <op-hw-jenkins+hostboot@us.ibm.com>
    Reviewed-by: Daniel M. Crowell <dcrowell@us.ibm.com>
```

Reply:
I'm all for just leaving P8 alone, but we'll take in a PR if someone really wants it.

Reply:
Thanks Dan.  Will keep this patch as is.